src/AnalysisConfigFiles/RecognizerConfigFiles/Towsey.NinoxStrenua.yml

```diff
@@ -0,0 +1,115 @@
+---
+
+# Powerful Owl = Towsey.NinoxStrenua
+
+# The parameters in this recognizer config have been tuned based on recordings provided by Kristen Thompson, DPI, NSW.
+# She suggests that separate recognizers should be prepared for male and female calls due to their difference in frequency.
+# The female calls seem pretty consistent between 450-550Hz. Often the first pulse is lower with the second pulse higher.
+# The male calls vary in frequency. Some are low at around 350Hz and don’t have the higher second pulse ...
+#    ... but some do and the highest that will go is 430Hz.
+
+
+# Resample rate must be 2 X the desired Nyquist
+#ResampleRate: 22050
+# SegmentDuration: units=seconds;    
+SegmentDuration: 60
+# SegmentOverlap: units=seconds;
+SegmentOverlap: 0
+
+# Each of these profiles will be analyzed
+# This profile is required for the species-specific recogniser and must have the current name.
+Profiles:
+    StrenuaSyllable6db: !ForwardTrackParameters
+        ComponentName: RidgeTrack 
+        SpeciesName: NinoxStrenua
+        FrameSize: 1024
+        FrameStep: 256
+        WindowFunction: HANNING
+        DecibelThreshold: 6.0
+        # min and max of the freq band to search
+        MinHertz: 300          
+        MaxHertz: 600
+        MinDuration: 0.2
+        MaxDuration: 0.9
+   
+    StrenuaSyllable12db: !ForwardTrackParameters
+        ComponentName: RidgeTrack 
+        SpeciesName: NinoxStrenua
+        FrameSize: 1024
+        FrameStep: 256
+        WindowFunction: HANNING
+        DecibelThreshold: 12.0
+        # min and max of the freq band to search
+        MinHertz: 300          
+        MaxHertz: 600
+        MinDuration: 0.2
+        MaxDuration: 0.9
+        
+    StrenuaSyllable24db: !ForwardTrackParameters
+        ComponentName: RidgeTrack 
+        SpeciesName: NinoxStrenua
+        FrameSize: 1024
+        FrameStep: 256
+        WindowFunction: HANNING
+        DecibelThreshold: 24.0
+        # min and max of the freq band to search
+        MinHertz: 300          
+        MaxHertz: 600
+        MinDuration: 0.2
+        MaxDuration: 0.9
+      
+    StrenuaSyllable30db: !ForwardTrackParameters
+        ComponentName: RidgeTrack 
+        SpeciesName: NinoxStrenua
+        FrameSize: 1024
+        FrameStep: 256
+        WindowFunction: HANNING
+        DecibelThreshold: 30.0
+        # min and max of the freq band to search
+        MinHertz: 300          
+        MaxHertz: 600
+        MinDuration: 0.2
+        MaxDuration: 0.9
+      
```

Comment:
There is quite some redundancy here. 
Use yaml backreferences?

Also, I wonder if forward track parameters could accept a range of decibels to inspect? Since that's the only value that's changing here.